h/static/styles/topbar.scss

```diff
@@ -41,7 +41,7 @@
 
     & > *:first-child, & > *:last-child {
       line-height: 28px;
-      margin: 0 .72em;
+      margin-right: .72em;
```

Comment:
This doesn't seem right. It feels like, if we want to have no margin on the share button, that this should be an override done by the share button. The default for the topbar (on the stream and other pages) is definitely to have symmetric margins.